content/en/docs/started/installing-kubeflow.md

```diff
@@ -9,148 +9,242 @@ Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it prov
 Operators can choose what is best for their users, there is no requirement to deploy every component.
 To read more about the components and architecture of Kubeflow, please see the <a href="/docs/started/architecture/">Kubeflow Architecture</a> page.
 
+### How to install Kubeflow?
+
 There are two pathways to get up and running with Kubeflow, you may either:
-1. Use a [packaged distribution](#packaged-distributions)
-1. Use the [manifests](#manifests) (advanced)
+
+1. [Use a packaged distribution](#packaged-distributions-of-kubeflow)
+2. [Use the raw manifests](#raw-kubeflow-manifests) <sup>(for advanced users)</sup>
 
 <a id="packaged-distributions"></a>
-## Install a packaged Kubeflow distribution
+<a id="install-a-packaged-kubeflow-distribution"></a>
+## Packaged Distributions of Kubeflow 
 
-{{% alert color="warning" %}}
-Packaged distributions are developed and supported by their respective maintainers, the Kubeflow community does not currently endorse or certify any distribution.
+{{% alert title="Note" color="warning" %}}
+Packaged distributions are developed and supported by their respective maintainers, <b>the Kubeflow community does not endorse or certify any specific distribution</b>.
+
+In the near future, there are plans to introduce <a href="https://github.com/kubeflow/community/blob/master/proposals/kubeflow-conformance-program-proposal.md">conformance testing for distributions</a>, you may track progress on this initiative by following <a href="https://github.com/kubeflow/kubeflow/issues/6485">kubeflow/kubeflow#6485</a>.
 {{% /alert %}}
 
-<b>See the table below for a list of options and links to documentation:</b>
+### Active Distributions
+
+The following table lists <b>active distributions</b> that have <b>had a recent release</b> (within the last 6 months).
```

Comment:
How about 12 months? especially with the [1.7 release timeline](https://github.com/kubeflow/community/tree/master/releases/release-1.7) starting early September and plan to release by March, by the time 1.7 is released, every distribution would be "legacy" if not updated to latest version.

Reply:
> How about 12 months? especially with the [1.7 release timeline](https://github.com/kubeflow/community/tree/master/releases/release-1.7) starting early September and plan to release by March, by the time 1.7 is released, every distribution would be "legacy" if not updated to latest version.

@annajung We probably should aim to push releases more frequently, lol!

I don't mind updating it to say 12 months, but there is a risk that 12 months of inactivity becomes acceptable (once we are releasing more frequently) if that's what the website says.